package sec02.Final;

public class test {
	//1. 3
	//2. 4. 새로운 예외를 발생시키기 위해서는 throw 를 사용하면 된다.
	//3. 3 중간에 Exception e 가 아래로 가야됨 
	//4. 
//	10
//	"숫자로 변활 수 없음"
//	10  finally 땜에
//	"인덱스를 초과했음"
//	10
	
}
